hasher-matcher-actioner/hmalib/scripts/cli/write_terraform_example.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import argparse
+import hcl2
+
+import hmalib.scripts.cli.command_base as base
+
+MAX_LINE_LENGTH = 80
+
+
+class WriteTerraformExampleConfigCommand(base.Command):
+    """
+    Takes the root input variables declaration file in
+    hasher-matcher-actioner/teraform directory and writes an example file
+    complete with description and defaults.
+
+    Input file is hardcoded. Output is to stdin.
+    """
+
+    @classmethod
+    def init_argparse(cls, ap: argparse.ArgumentParser) -> None:
+        return super().init_argparse(ap)
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "write-terraform-example"
```

Comment:
nit/unsure: naming is hard but this feels like it is a little too ambiguous thoughts on `gen-tfvars-exanple` or `print-tfvars-example`